docs/advanced/images-adapters/cloudinary.mdx

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 1
+title: Cloudinary
+description:
+  Streamline media management and improve user experience by automatically
+  delivering images enhanced and optimized for every user.
+tags:
+  - Image Adapter
+  - Cloudinary
+  - Since version 2.20.x
+---
+
+import ContactLink from "@site/src/components/ContactLink";
+
```

Comment:
The `<SinceVersion>` component must be used here to mention 2.20

Reply:
Will do, see https://github.com/front-commerce/developers.front-commerce.com/pull/555#discussion_r1041094217